client/src/pages/dashboard/Dashboard.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Grid } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+import Navbar from '../layout/Navbar';
+
+const useStyles = makeStyles(theme => ({
+    sidebar: {
+        height: '85vh',
+        backgroundColor: '#f9f9fc',
+        alignItems: 'center',
+        flexGrow: 1,
+        marginTop: 3,
+    },
+    content_page: {
+        backgroundColor: theme.palette.common.white,
+        marginTop: 3,
+    },
+    profile: {
+        flexGrow: 2,
+    },
+    list: {
+        flexGrow: 6,
+    },
+    logout: {
+        flexGrow: 1,
+    },
+}));
+
+const Dashboard = () => {
+    const classes = useStyles();
+    return (
+        <Grid>
+            <Navbar />
+            <Grid container>
+                <Grid
```

Comment:
You can basically wrap up this skeleton by moving Sidebar out into its own component!